HISTORY.rst

```diff
@@ -6,12 +6,19 @@ latest
 ------
 
 Minor changes:
-
+~~~~~~~~~~~~~
 * Added the flag ``--mca btl_vader_single_copy_mechanism none to mpirun in fv3run`` to mpirun in fv3run
 * Add ReadTheDocs configuration file
 * Do not require output dir and fv3config to be remote in ``run_kubernetes``
 * Fix bug when submitting k8s jobs with images that have an "_" in them
 
+Breaking changes
```

Comment:
We've been putting these under "Major changes" in fv3gfs-python, but I don't object to this heading.